configs/default/BEFH-BETAFPVF411RX.config

```diff
@@ -0,0 +1,100 @@
+# Betaflight / STM32F411 (S411) 4.1.0 Jun 17 2019 / 23:11:43 (f7f0c6780) MSP API: 1.42
+
+board_name BETAFPVF411RX
+manufacturer_id BEFH
+
+# resources
+resource MOTOR 1 B08
+resource MOTOR 2 B01
+resource MOTOR 3 B10
+resource MOTOR 4 B07
+resource PPM 1 A10
+resource LED_STRIP 1 A00
+resource SERIAL_TX 1 A09
+resource SERIAL_TX 2 A02
+resource SERIAL_TX 11 A08
+resource SERIAL_RX 1 A10
+resource SERIAL_RX 2 A03
+resource SERIAL_RX 11 A08
```

Comment:
You have softserial1 RX and TX assigned to the same pin.

Reply:
This is not wrong per se - configured this way a softserial with only a single pin available can be used for TX, RX, and half duplex mode.

Reply:
If it’s intentional I guess. But would seem to be very confusing for the user. And common softserial uses like GPS won’t work.

Reply:
GPS should work, just in NMEA mode only. I don't think there is a way to make ublox mode work with an UART with only one pin available, so not defining `resource SERIAL_TX 11` will not improve the situation.
Similar settings are already used by existing targets (https://github.com/betaflight/unified-targets/blob/11c9eb43d7b4bdaa11920d671760d51aa869c7e2/configs/default/AIRB-NOX.config#L15-L18), so this is probably just a case of copy / pasta.

Reply:
softserial1 RX and TX For VTX control,UART1 and UART2 are complete, they can be used for camera or GPS.